src/System.Text.Json/tests/JsonStringTests.cs

```diff
@@ -0,0 +1,128 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.Text.Json.Tests
+{
+    public static class JsonStringTests
+    {
+        [Fact]
+        public static void TestDefaultCtor()
+        {
+            var jsonString = new JsonString();
+            Assert.Equal("", jsonString.Value);
+        }
+
+        [InlineData("value")]
+        [InlineData("value with some spaces")]
+        [InlineData("     training spaces")]
+        [InlineData("new lines\r\n")]
+        [InlineData("tabs\ttabs\t")]
+        [InlineData("\\u003e\\u003e\\u003e\\u003e\\u003e")]
+        [InlineData("zażółć gęślą jaźń")]
+        [InlineData("汉字 漢字")]
```

Comment:
We generally avoid non-ascii text within the tests.

Use the \\uXXXX format to represent these characters (same elsewhere).

So, in this case:
```suggestion
        [InlineData("\u6f22\u5b57 \u6f22\u5b57")]
```

Eg:
https://github.com/dotnet/corefx/blob/d3911035f2ba3eb5c44310342cc1d654e42aa316/src/System.Text.Json/tests/Utf8JsonReaderTests.cs#L3874
https://github.com/dotnet/corefx/blob/d3911035f2ba3eb5c44310342cc1d654e42aa316/src/System.Text.Json/tests/Serialization/Value.WriteTests.cs#L16

Reply:
Quoting @danmosemsft (from https://github.com/dotnet/corefx/pull/33216#discussion_r230973414),
> Please don't rely on the encoding of this file not changing. Historically given enough time, it will get translated to ASCII somehow. We see this in old .NET Framework test files. Sometimes nobody notices because the test still passes (both the input and expected got garbled the same)

> From coding-style.md:
`When including non-ASCII characters in the source code use Unicode escape sequences (\uXXXX) instead of literal characters. Literal non-ASCII characters occasionally get garbled by a tool or editor.`